docs/content/latest/yugabyte-platform/secure-universes/enable-encryption-at-rest.md

```diff
@@ -0,0 +1,122 @@
+
+1. Open the Yugabyte Platform console and go to the universe.
+2. Select the **Nodes** tab.
+3. On one of the nodes, click **Master** under the **PROCESSES** column. The overview page appears.
+4. To the right of **Replication Factor**, click **See full config**. The **Current Cluster Config** page appears.
+5. Verify that the configuration includes the following `encryption_info` section with the correct values:
+
+    ```json
+    encryption_info {
+      			encryption_enabled: true,
+    Universe_key_registry_encoded: ".*",
+    Key_in_memory: true,
+    latest_version_id: ".*",
+    }
+    ```
+
+6. Go to the **Security Object** page for the Equinix SmartKey account that was used to create the KMS configuration. A security object should exist with a name matching the universe UUID that was configured to use encryption at rest with the SMARTKEY configuration (defaults to AES 256).
+
+**Equinix SmartKey**
+
+Once the universe has started being created, a security object will be created with SmartKey through REST API with the universe UUID as the name of the security object. This object will then be exported to Yugaware as the universe key. Once received, Yugaware will then send the key to all masters to persist. Yugaware does not persist any information about the security object beyond a reference to the object’s kId. At this point, data within the universe will be encrypted at rest using data keys encrypted with the aforementioned universe key generated through the chosen KMS configuration.
+
+**AWS KMS**
+
+Once the universe has been created with encryption at rest enabled, the platform will persist the ciphertext of the universe key (because AWS does not persist any CMK-generated data keys themselves) and will request the plaintext of the universe key from AWS KMS using the KMS configuration whenever it needs to provide the universe key to the master nodes.
+
+## Enable encryption at rest on an existing universe
+
+To enable encryption at rest on an existing universe, follow these steps:
+
+1. Open the Yugabyte Platform console and click **Universes**.
+2. Select the universe you want to enable.
+3. Click the **More** drop-down list and select **Manage Encryption Keys**. The **Manage Keys** dialog appears.
+4. Select **Enable Encryption-at-Rest for <UNIVERSE_NAME>**. When enabled, the **Key Management Service Config** option appears.
+5. Select your KSM configuration from the **Key Management Service Config** drop-down list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration](../create-kms-configuration).
+6. Verify that the `EnableEncryptionAtRest` task completed successfully.
+
+## Back up and restore data from an encrypted at rest universe
+
+When backing up a cluster, you might restore the data to a different cluster with a different universe key or it might not have encryption enabled. As a result, a backup must include the decrypted data keys for all files.
```

Comment:
I don't think a backup currently includes decrypted data keys. Since the data files are backed up "as is", they remain encrypted in the file header.

Reply:
to follow up, we include the metadata file that includes only a list of key id's for every universe key in the registry. That is enough for us to get the key values in the restore flow (as long as the user selects an appropriate KMS configuration in the restore backup modal)

Reply:
I'll fix that -- that wording was from the design doc. I guess that should be updated too.